gadopt/level_set_tools.py

```diff
@@ -67,7 +67,13 @@ def interface_thickness(
       A Firedrake function holding the interface thickness values
     """
     epsilon = fd.Function(level_set_space, name="Interface thickness")
-    epsilon.interpolate(scale * fd.MinCellEdgeLength(level_set_space.mesh()))
+
+    if level_set_space.extruded:
+        scale = scale or 0.25
```

Comment:
Why are these values different (0.25 for extruded versus 0.35 for other)? Sorry if it's obvious... but it wasn't to me from quick thinking.

Reply:
Because `0.25 * sqrt(2)` is roughly 0.35. See also my comment in the main thread.

Reply:
I think magic numbers like this deserve a comment so the history isn't lost to a GitHub thread!